frontend/app.js

```diff
@@ -34,26 +34,36 @@ App = function () {
     // Dynamic Canvas Sizes
     var elmntmjpegRightForearm = document.getElementById("camera1");
     var rightForearmWidth=elmntmjpegRightForearm.clientWidth;
-    var rightForearmHeight=elmntmjpegRightForearm.clientHeight;
+    var rightForearmHeight=rightForearmWidth;
```

Comment:
I think we actually need the view to be 480px by 640px, or some multiple of those, because that is the size of the camera frame on the back end.

Reply:
for dynamic changes of the browser size?